Rakefile

```diff
@@ -13,7 +13,7 @@ Hoe.spec 'net-http-persistent' do
   self.readme_file      = 'README.rdoc'
   self.extra_rdoc_files += Dir['*.rdoc']
 
-  self.require_ruby_version '~> 2.3'
+  self.require_ruby_version '>= 2.3'
```

Comment:
I'm just curious where this method(`require_ruby_version=`) comes from.

In the docs and repo of rubygems I found only `required_ruby_version=` https://guides.rubygems.org/specification-reference/#required_ruby_version=
https://github.com/rubygems/rubygems/search?q=required_ruby_version

https://github.com/rubygems/rubygems/search?q=require_ruby_version

Reply:
This gem uses @seattlerb's `hoe` to build the gemspec: https://github.com/seattlerb/hoe/search?q=require_ruby_version

Reply:
Ah, got it. I didn't notice the this is Rakefile. :smile: